build

```diff
@@ -124,4 +124,9 @@ if [ -d cert ]; then
 	container_mount_opts+=(-v "$PWD/cert:/builder/cert:ro")
 fi
 
+if [ "$container_engine" = "docker" ] && out=$(sysctl kernel.apparmor_restrict_unprivileged_userns) && [[ $out = "kernel.apparmor_restrict_unprivileged_userns = 1" ]]; then
```

Comment:
how about non-apparmor systems (like fedora)? how about docker desktop?

will this condition be false on those systems and just be a no-op?